src/services/queries/auth/useSignin.ts

```diff
@@ -34,18 +46,25 @@ const useAdminSignin = () => {
           // secure: true,
           // httpOnly: true,
         });
-        // eslint-disable-next-line no-void, @typescript-eslint/no-floating-promises
-        getArtistNames();
         dispatch(setUser(data.member));
-        // eslint-disable-next-line @typescript-eslint/no-floating-promises
-        router.push(`/admin/dashboard/${getLatestYearMonthString()}`);
+        if (data.member.role === "PENDING") {
```

Comment:
미리 만들어둔 상수를 사용하면 좋을 것 같습니다!
```suggestion
        if (data.member.role === REQUEST_AUTHORITY_STATUS.PENDING) {
```

Reply:
넵넵